Packs/Salesforce/Integrations/SalesforceV2/SalesforceV2.py

```diff
@@ -0,0 +1,1041 @@
+        "Priority": args.get('priority'),
+        "Type": args.get('caseType')
+    }
+
+    results = client.createObject('Case', data)
+    if results.get('success', None):
+        case = client.getObject(f"{results.get('id')}")
+    else:
+        return_results(f"Unable to create case. Error Encountered was:{json.dumps(results.get('errors', None))}")
+
+    return objectToEntry(client, case)
+
+
+def update_case_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    if args.get('oid', None) and args.get('caseNumber', None):
+        return_error('You must specify object ID or a Case Number')
+
+    if args.get('caseNumber', None):
+        condition = f"CaseNumber='{args.get('caseNumber')}'"
+        cases = client.queryObjects(['Id'], 'Case', condition).get('records')
+        if len(cases) > 0:
+            oid = cases[0].get('Id')
+        else:
+            return_error('Unable to update case -> Invalid Case Number provided')
+    else:
+        oid = args.get('oid')
+
+    data = {}
+
+    if 'subject' in args:
+        data['Subject'] = args.get('subject')
+
+    if 'description' in args:
+        data['Description'] = args.get('description')
+
+    if 'status' in args:
+        data['Status'] = args.get('status')
+
+    if 'origin' in args:
+        data['Origin'] = args.get('origin')
+
+    if 'priority' in args:
+        data['Priority'] = args.get('priority')
+
+    if 'caseType' in args:
+        data['Type'] = args.get('caseType')
+
+    caseUpdate = client.updateObject(f'Case/{oid}', data)
+
+    return get_case_command(client, {'oid': oid})
+
+
+def get_cases_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    properties = ['ID', 'CaseNumber', 'Subject', 'Description', 'CreatedDate',
+                  'ClosedDate', 'OwnerID', 'Priority', 'Origin', 'Status', 'Reason']
+    cases = client.queryObjects(properties, 'Case', '').get('records')
+    return objectToEntry(client, cases)
+
+
+def close_case_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    args['status'] = 'Closed'
+    return update_case_command(client, args)
+
+
+def delete_case_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    if args.get('oid', None) and args.get('caseNumber', None):
+        return_error('You must specify object ID or a Case Number')
+
+    if args.get('caseNumber', None):
+        condition = f"CaseNumber='{args.get('caseNumber')}'"
+        cases = client.queryObjects(['Id'], 'Case', condition).get('records')
+        if len(cases) > 0:
+            oid = cases[0].get('Id')
+        else:
+            return_error('Unable to update case -> Invalid Case Number provided')
+    else:
+        oid = args.get('oid')
+
+    return client.deleteObject(f'Case/{oid}')
+
+
+def push_comment_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    return client.pushComment(args.get('oid'), args.get('text'), args.get('linkUrl'))
```

Comment:
Please return the response as a CommandResults object.

Reply:
This comment is not relevant anymore because the Pack's support level has changed to - 'community'.